packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1017,6 +1017,14 @@ abstract class State<T extends StatefulWidget> with Diagnosticable {
   /// return a future (the callback cannot be `async`), since then it would be
   /// unclear when the state was actually being set.
   ///
+  /// In fact, this callback is a simple convention that allows anyone reading
```

Comment:
per our style guide, you can avoid the "In fact" (see "empty prose" in the style guide)